src/main.rs

```diff
@@ -158,66 +109,44 @@ async fn main() -> Result<(), anyhow::Error> {
     let meta = package.metadata;
     debug!("Retrieved metadata: {:?}", meta);
 
-    // Select which binary path to use
-    let pkg_url = match (opts.pkg_url, meta.as_ref().map(|m| m.pkg_url.clone() ).flatten()) {
-        (Some(p), _) => {
-            info!("Using package url override: '{}'", p);
-            p
-        },
-        (_, Some(m)) => {
-            info!("Using package url: '{}'", &m);
+    // Select which package path to use
+    let pkg_url = match meta.as_ref().map(|m| m.pkg_url.clone() ).flatten() {
+        Some(m) => {
+            debug!("Using package url: '{}'", &m);
             m
         },
         _ => {
-            info!("No `pkg-url` key found in Cargo.toml or `--pkg-url` argument provided");
-            info!("Using default url: {}", DEFAULT_BIN_PATH);
-            DEFAULT_BIN_PATH.to_string()
+            debug!("No `pkg-url` key found in Cargo.toml or `--pkg-url` argument provided");
+            debug!("Using default url: {}", DEFAULT_PKG_PATH);
+            DEFAULT_PKG_PATH.to_string()
         },
     };
 
     // Select bin format to use
-    let pkg_fmt = match (opts.pkg_fmt, meta.as_ref().map(|m| m.pkg_fmt.clone() ).flatten()) {
-        (Some(o), _) => o,
-        (_, Some(m)) => m.clone(),
+    let pkg_fmt = match meta.as_ref().map(|m| m.pkg_fmt.clone() ).flatten() {
```

Comment:
this isn't a _default_ because the `meta` object may be `None`, probably should be replaced with `unwrap_or(Meta::default())` and `#[serde(default)]`